Make installation of dev-lang/vala bindings optional. Unfortunately
the option name "vala" is reserved so we must use a different one.

--- a/libjcat/meson.build
+++ b/libjcat/meson.build
@@ -122,6 +122,7 @@
     install : true
   )
 
+  if get_option('vala-bindings')
   gnome.generate_vapi('jcat',
     sources : jcat_gir[0],
     packages : [
@@ -130,6 +131,7 @@
     ],
     install : true,
   )
+  endif
 
   python = import('python')
   python_interpreter = python.find_installation('python3',
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -4,3 +4,4 @@
 option('gpg', type : 'boolean', value : true, description : 'enable the GPG verification support')
 option('pkcs7', type : 'boolean', value : true, description : 'enable the PKCS7 verification support')
 option('man', type : 'boolean', value : true, description : 'enable man pages')
+option('vala-bindings', type : 'boolean', value : true, description : 'enable vala bindings')
